### PR TITLE
Make UnsafeUtils public

### PR DESCRIPTION
**Pull request by liran-funaro**
This PR merges the changes in 0.2.3.1 to master.

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: core/src/main/java/com/yahoo/oak/DirectUtils.java [hunk 1]
```diff
@@ -13,26 +13,32 @@
 import java.nio.Buffer;
 import java.nio.ByteBuffer;
 
-final class UnsafeUtils {
+public final class DirectUtils {
```
Comment (sanastas) [inline core/src/main/java/com/yahoo/oak/DirectUtils.java:16]: As this is public, I suggest to add some comments here about the usage and also about Java version required. Maybe some very short mentioning in the README as well.

**Review by sanastas — APPROVED** [review]
Looks good, small comment below.

File: core/src/main/java/com/yahoo/oak/DirectUtils.java (hunk 1 above)
Comment (liran-funaro) [reply core/src/main/java/com/yahoo/oak/DirectUtils.java:16]: To the best of my knowledge, this should work on any version as long as Oak was compiled on Java8.

Comment (sanastas) [reply core/src/main/java/com/yahoo/oak/DirectUtils.java:16]: So just add a comment about it, some users may question about the possible usage

Comment (liran-funaro) [reply core/src/main/java/com/yahoo/oak/DirectUtils.java:22]: I added documentation

**Pull request merged**